relay-server/src/metrics_extraction/event.rs

```diff
@@ -12,28 +12,62 @@ use relay_sampling::FieldValueProvider;
 /// extracted. Timestamp and clock drift correction should occur before metrics extraction to ensure
 /// valid timestamps.
 pub fn extract_event_metrics(event: &Event, config: &MetricExtractionConfig) -> Vec<Metric> {
+    extract_metrics_from(event, config)
+}
```

Comment:
Now that we can use `extract_metrics_from`, is `extract_event_metrics` useful?

Reply:
Good catch, I removed it and renamed the module to `generic`.